Source/Relativity.Test.Helpers/Relativity.Tests.Helpers.Tests.Unit/ArtifactHelpers/DocumentHelperTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using Relativity.Test.Helpers;
+using Relativity.Test.Helpers.ArtifactHelpers;
+using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
+using TestHelpersKepler.Interfaces.TestHelpersModule.v1.Models;
+
+namespace Relativity.Tests.Helpers.Tests.Unit.ArtifactHelpers
+{
+	[TestFixture]
+	public class DocumentHelperTest
```

Comment:
it makes sense to rename to DocumentHelperTests

Reply:
Done